torch/csrc/distributed/rpc/tensorpipe_agent.cpp

```diff
@@ -716,7 +769,10 @@ std::shared_ptr<JitFuture> TensorPipeAgent::send(
         pipeRead(
             clientPipe.pipe_,
             [this, &clientPipe](
-                const tensorpipe::Error& error, Message&& responseMessage) {
+                const tensorpipe::Error& error,
+                Message&& responseMessage,
+                // NOLINTNEXTLINE(performance-unnecessary-value-param)
```

Comment:
Is this `NOLINT` still needed now that we forward the ctx to `markFutureAsComplete`?